tools/build_pytorch_libs.bat

```diff
@@ -10,7 +10,14 @@ set INSTALL_DIR=%cd:\=/%/torch/lib/tmp_install
 set THIRD_PARTY_DIR=%cd:\=/%/third_party
 set BASIC_C_FLAGS=
 set BASIC_CUDA_FLAGS=
-set LDFLAGS=/LIBPATH:%INSTALL_DIR%/lib
+
+IF NOT DEFINED CMAKE_INSTALL_PREFIX (
```

Comment:
Why aren't you just able to set INSTALL_DIR instead of CMAKE_INSTALL_PREFIX? Otherwise I think you'd need to update build_pytorch_libs.sh to match here.